test/WebJobs.Script.Tests/WebApiApplicationTests.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Linq;
+using Microsoft.Azure.WebJobs.Script.WebHost;
+using Xunit;
+
+namespace Microsoft.Azure.WebJobs.Script.Tests
+{
+    public class WebApiApplicationTests
+    {
+        [Theory]
+        [InlineData(@"c:\somedirectory", @"c:\testpath")]
+        [InlineData(@"c:\somedirectory", "")]
+        [InlineData(@"c:\somedirectory", null)]
+        [InlineData("", @"c:\testpath")]
+        [InlineData(null, @"c:\testpath")]
+        public void GetShadowCopyPath_ReturnsExpectedPath(string currentShadowCopy, string scriptPath)
+        {
+            string updatedPath = WebApiApplication.GetShadowCopyPath(currentShadowCopy, scriptPath);
+            string expectedPath = string.Join(";", new[] { currentShadowCopy, scriptPath }.Where(s => !string.IsNullOrEmpty(s)));
```

Comment:
Can you manually put the expected path as an input? Otherwise you're verifying the same code here. That'd make it clearer exactly what you expect from the method.